dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -405,6 +408,21 @@ String getSql(@Nonnull Predicate<? super String> predicate) {
             return getSql(predicate, EXPRESSIONS_FOR_ASSESSMENT);
         }
     }
+
+    private ImmutableList<TeradataJdbcSelectTask> createTimeSeriesTasks(ZonedInterval interval) {
+        return ImmutableList.of("ResUsageScpu","ResUsageSpma").stream().map(tableName ->
+            new TeradataJdbcSelectTask(createFilename("dbc."+tableName+"_", interval), TaskCategory.OPTIONAL,
+            String.format("SELECT %%s FROM DBC.%s WHERE TheTimestamp >= %s AND TheTimestamp < %s",
```

Comment:
The string literal here is a pattern for the `String#format` method, which itself produces a pattern that is then passed to the `TeradataJdbcSelectTask` and another `String#format` invocation is executed there. I agree it's a bit convoluted at the moment. We are working on improving the SQL generation logic, so hopefully this will be simplified in another PR.